benchto-driver/src/main/java/com/teradata/benchto/driver/graphite/GraphiteMetricsLoader.java

```diff
@@ -78,14 +78,19 @@ public void initQueryMetrics()
             return emptyList();
         }
 
-        long cutOffThresholdSeconds = executionSynchronizer.cutOffThresholdSecondsForMeasurementReporting();
+        /*
+         * Graphite can be queried with seconds resolution. It's `from` is exclusive (sic!, at least currently) but `until` is inclusive.
+         * Subtracting graphite resolution from `until` we effectively ask for the buckets that are fully covered by the `measurable`.
+         * This ignores first and last bucket (both partially covered), but gives most interesting statistics.
+         */
+        long fromEpochSecond = measurable.getUtcStart().toEpochSecond();
+        long toEpochSecond = measurable.getUtcEnd()
```

Comment:
If it ended at `4.56`, we would truncate it to `4`, then subtract resolution and get `3`, so we will ask for all "buckets" until (inclusive) `3`. But "bucket" `3` covers range `[3, 4)`, so it will be (a) wholly covered by execution and (b) last one such.